### ISSUE TITLE
Set up email sub domains

### ISSUE THREAD

**Issue by lauracumming** [ocadotechnology/codeforlife-admin#1107]
**Summary**
We are having some bounce backs with emails from Dotdigital.

**User story**
_As a user
_I would like to receive the emails i need 
_so that I can carry out the tasks

**Outcome/deliverable**
Set up 3 subdomains for a) transactional, b) marketing, c) reach

**Deadline**
TBC

**Links**
Insert any hyperlinks to sites or documents that may help with this task.

**Reviewer**
Who should review this task?

**People to contact**
https://support.dotdigital.com/en/articles/8199003-branded-domains-sending-with-a-custom-from-address-cfa

**Comment by lauracumming** [comment]
Waiting on PO. Purchasing one subdomain to test.

**Comment by lauracumming** [comment]
PO raised and sent

**Comment by lauracumming** [comment]
This is now activated - I will add a new task for set up
